ignite/engine/engine.py

```diff
@@ -321,6 +323,23 @@ def remove_event_handler(self, handler, event_name):
             raise ValueError("Input handler '{}' is not found among registered event handlers".format(handler))
         self._event_handlers[event_name] = new_event_handlers
 
+    @staticmethod
```

Comment:
We have now both `_check_signature` in Engine and in utils.py.

Reply:
Oh, sorry. Must have missed that during the rebasing. Will fix this!